packages/editor/src/components/list-view-sidebar/style.scss

```diff
@@ -22,11 +22,21 @@
 	// This allows items in the list view to have equidistant padding left and right
 	// right up until a scrollbar is present.
 	scrollbar-gutter: auto;
+}
 
-	// The table cells use an extra pixels of space left and right. We compensate for that here.
+// The table cells use an extra pixels of space left and right. We compensate for that here.
+.editor-list-view-sidebar__list-view-panel-content {
+	// Keep the original padding for the List View panel so its spacing matches trunk.
 	padding: $grid-unit-05;
 }
 
+// Apply the new padding only to the Document Outline panel.
+.editor-list-view-sidebar__list-view-container > .document-outline {
+	// Override the global top margin (20px) so it doesn’t stack with the new padding.
+	margin-top: 0;
+	padding: $grid-unit-20;
+}
```

Comment:
We may want to do this directly in the document [outline component](https://github.com/WordPress/gutenberg/blob/trunk/packages/editor/src/components/document-outline/style.scss#L5). I can't think of reasons to override the styles only in the list view.

Are you using the child combinator (`>`) for any reason in particular here?